ui/pages/settings/index.js

```diff
@@ -299,6 +315,17 @@ export default function Settings() {
                     ),
                     accessorKey: 'created',
                   },
+                  {
+                    cell: info => (
+                      <div className='hidden sm:table-cell'>
+                        {info.getValue() ? moment(info.getValue()).from() : '-'}
+                      </div>
```

Comment:
and this is how we handle when its on the bigger screen where we see the value in the table - so if the value is not set then it will return `-`